pytensor/tensor/slinalg.py

```diff
@@ -380,6 +382,186 @@ def cho_solve(c_and_lower, b, *, check_finite=True, b_ndim: int | None = None):
     )(A, b)
 
 
+class LU(Op):
```

Comment:
Missing gufunc_spec or whatever it's called, so that the Blockwise impl can avoid vectorize as scipy LU already works with batch dims?